docs/src/api.md

```diff
@@ -6,62 +6,4 @@
 ```@autodocs
 Modules = [PiCraft]
 Order   = [:type, :function]
-```
```

Comment:
autodocs is working now?

Reply:
yes. It dumps all commands together. I'll tidy it up soon by grouping relevant commands together.